velox/type/Filter.h

```diff
@@ -1262,6 +1269,16 @@ class MultiRange final : public Filter {
   std::unique_ptr<Filter> clone(
       std::optional<bool> nullAllowed = std::nullopt) const final;
 
+  const std::vector<std::unique_ptr<Filter>>& filters() const {
```

Comment:
Any particular reason for moving these methods? It is hard to review when moves are mixed with material changes.

Reply:
They are not moves exactly. I had them in one place but #299 was pushed which included the same functions in a different location, hence it shows up this way on github